frontend/src/citizen-frontend/messages/ThreadView.tsx

```diff
@@ -203,6 +203,7 @@ interface Props {
   thread: CitizenMessageThread.Regular
   closeThread: () => void
   onThreadDeleted: () => void
+  setInitFocusFunc: (initFocusFunc: () => void) => void
 }
 
 export default React.memo(function ThreadView({
```

Comment:
ThreadView muutetaan React.forwardRef:ksi
```suggestion
export default React.memo(
  React.forwardRef<HTMLDivElement, Props>(function ThreadView(
    {
      accountId,
      thread: {
        id: threadId,
        messages,
        title,
        messageType,
        urgent,
        sensitive,
        children
      },
      closeThread,
      onThreadDeleted
    }: Props,
    ref
  ) {
```